### DISCUSSION TITLE
Why are PRs not being merged that have been open more than 3 months?

### DISCUSSION THREAD

**Discussion by doodle-codes** [gluestack/gluestack-ui#2208] [General]
Dear Gluestack Admin Team,

I am writing to express how frustrated and concerned we are about several PRs that have been open and unmerged for over three months in the Gluestack repository. These prolonged delays are causing significant impediments to our development process and affecting our team's productivity and efficiency.

Many of these PRs contain critical fixes and improvements that our projects depend on. The extended waiting period for these PRs to be reviewed and merged is causing bottlenecks, leading to stalled progress and increased technical debt. Our team relies heavily on these updates to maintain and enhance our applications, and the current limbo is unsustainable.

Please prioritize the review and merging of these pending PRs. Timely action on these contributions is essential for our continued success and the overall health of the Gluestack ecosystem. We appreciate the hard work and dedication of the admin team and hope for a swift resolution to this pressing issue.

Thank you for your attention and understanding.

**Comment by Viraj-10** [comment]
Hey @doodle-codes, Can you share PR link?

**Reply by doodle-codes** [reply]
Here are some of the PRs we would like to see merged:
- https://github.com/gluestack/gluestack-ui/pull/1840 <- At least merge this component fix or have it finalized.
- https://github.com/gluestack/gluestack-ui/pull/1785
- https://github.com/gluestack/gluestack-ui/pull/1735
- https://github.com/gluestack/gluestack-ui/pull/1588
- https://github.com/gluestack/gluestack-ui/pull/1571

**Comment by evelant** [comment]
I added Bun support for the v2 cli, would appreciate if the PR got a look https://github.com/gluestack/gluestack-ui-cli/pull/83
